fulltext/services/tests/test_pdf.py

```diff
@@ -0,0 +1,147 @@
+"""Tests for :mod:`fulltext.services.pdf`."""
```

Comment:
Since the responses are mocked I'm not sure how much varying the ID format in this particular suite will exercise things. But I did add some more tests in a52000ac (additional PDFs, varying ID types) and 2af7977a (store and retrieve with varying ID types)